doc/developers/contributing.rst

```diff
@@ -934,8 +934,9 @@ Monitoring performance
 When proposing changes to the existing code base, it's important to make sure
 that they don't introduce performance regressions. Scikit-learn uses
 `asv benchmarks <https://github.com/airspeed-velocity/asv>`_ to monitor the
-performance of a selection of common estimators and functions. The benchmark
-suite can be found in the `scikit-learn/asv_benchmarks` directory.
+performance of a selection of common estimators and functions, you can view 
+these benchmarks on the `scikit-learn benchmark page <https://scikit-learn.org/scikit-learn-benchmarks>`_. 
+The benchmark suite can be found in the `scikit-learn/asv_benchmarks` directory.
```

Comment:
```suggestion
The corresponding benchmark suite can be found in the `scikit-learn/asv_benchmarks` directory.
```

Reply:
This makes things a lot clearer, thanks! Added as well.